src/pages/task/actionButtons/relevantCommits/index.tsx

```diff
@@ -85,6 +90,31 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
     },
   });
   const lastPassingTask = getTaskFromMainlineCommitsQuery(lastPassingTaskData);
+  const passingOrderNumber =
+    getOrderFromMainlineCommitsQuery(lastPassingTaskData);
+
+  const { data: breakingTaskData, loading: breakingLoading } = useQuery<
+    LastMainlineCommitQuery,
+    LastMainlineCommitQueryVariables
+  >(LAST_MAINLINE_COMMIT, {
+    skip:
+      !parentTask ||
+      !lastPassingTask ||
+      baseStatus === undefined ||
```

Comment:
Curious if this is needed! I feel like the task status should not be `undefined` but let me know if you encountered a scenario where it was 👀
```suggestion
```

Reply:
<img width="119" alt="Screenshot 2024-02-20 at 4 08 54 PM" src="https://github.com/evergreen-ci/spruce/assets/64446617/673bbbd2-b140-4741-adcf-5c593d2b7451">

It starts as undefined and then updates - but that's because the baseTask starts as undefined and an empty object is used.

I did some more testing and I think it is needed even though the other conditions usually cause this query to skip too but I'm only worried about a race condition- i.e. parentTask and lastPassingTask load before the current task.